### PR TITLE
Fix a problem with the entrypoint.sh vs entrypoint.dev.sh

### PR DESCRIPTION
**Pull request by reiterl**
In entrypoint.sh variables are exported. As it is started with 'bash'
they live in a different process. "source" doesn't work here, because
of the exec in the entrypoint.sh. So the only way seems to be to
double the code.

### CODE REVIEW

**Comment by reiterl** [comment]
Arg, I found a different problem with this approach. The testdata should not be loaded in the run-dev from the main repo.

**Comment by reiterl** [comment]
Add a conditional and a test env variable. So the test data is only inserted for the tests.

**Comment by GabrielInTheWorld** [comment]
Maybe this is a good help: https://stackoverflow.com/a/9772093

File: entrypoint.sh [hunk 1]
```diff
@@ -5,6 +5,14 @@ set -e
...
+then
+	MEDIA_DATABASE_USER=$(cat $MEDIA_DATABASE_USER_FILE)
+fi;
+if [ -n "$MEDIA_DATABASE_PASSWORD_FILE" ];
+then
+	MEDIA_DATABASE_PASSWORD=$(cat $MEDIA_DATABASE_PASSWORD_FILE)
+fi;
 export MEDIA_DATABASE_USER="${MEDIA_DATABASE_USER:-openslides}"
 export MEDIA_DATABASE_PASSWORD="${MEDIA_DATABASE_PASSWORD:-openslides}"
 PGPASSWORD="$MEDIA_DATABASE_PASSWORD"
```
Comment (GabrielInTheWorld) [inline entrypoint.sh:18]: Please try to separate these variables into a third shell-script and export them. Then, source this shell-script into the both entrypoints.

**Review by GabrielInTheWorld — CHANGES_REQUESTED** [review]

File: entrypoint.sh (hunk 1 above)
Comment (reiterl) [reply entrypoint.sh:18]: Mmh, if we use a different file, I have a different idea: I separate all vars in a file and call them with `source` in both entrypoints.

**Comment by reiterl** [comment]
Clean the `entrypoint.dev.sh`. It was a code doubling to the `entrypoint.sh` except the last call. This simplifies this.

**Review by GabrielInTheWorld — APPROVED** [review]
This works!

**Pull request merged**